build/appprotect/DockerfileWithAppProtectForPlus

```diff
@@ -92,7 +94,7 @@ RUN  mkdir -p /var/lib/nginx \
 			    /var/log/nginx/ \
 	&& apt-get remove --purge -y libcap2-bin \
 	&& rm /etc/nginx/conf.d/*
-
+	
```

Comment:
whitespace alert :)

Reply:
all the RUN statements seem to be seperated by a whitespace from the others

Reply:
@rafwegv Right, but I think @pleshakov is saying it can just be an empty line, so the indentation can be removed just leaving an empty line remaining.

Reply:
ok i see it now.